source/en/common_structs.rst

```diff
@@ -188,3 +188,48 @@ Methods
   The common use case of this method is to close the TCP connection explicitly or to change the timeout.
 
   ``mprpc_client`` is a type of MessagePack-RPC client that is different between languages (`C++ <http://ci.jubat.us/job/msgpack-rpc/doxygen/classmsgpack_1_1rpc_1_1client.html>`_ / `Python <https://github.com/msgpack/msgpack-rpc-python/blob/master/msgpackrpc/client.py>`_ / `Ruby <http://msgpack.org/rpc/rdoc/current/MessagePack/RPC/Client.html>`_ / `Java <http://msgpack.org/rpc/javadoc/current/org/msgpack/rpc/Client.html>`_).
+
+
+Auxiliary Methods
+~~~~~~~~~~~~~~~~~
+
+Each language has some language-specific auxiliary methods.
```

Comment:
"Each language has some..." seems that all client languages have auxiliary methods.